src/components/buttonPanel.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Button from './buttons';
+
+export default class ButtonPanel extends React.Component {
+    renderButton = (i) => <Button name={i} />
+
+    render() {
+      return (
+        <div>
+          <div className="calculator-row">
+            {this.renderButton('AC')}
+            {this.renderButton('+/-')}
+            {this.renderButton('%')}
+            {this.renderButton('/')}
+          </div>
+          <div className="calculator-row">
+            {this.renderButton('7')}
+            {this.renderButton('8')}
+            {this.renderButton('9')}
+            {this.renderButton('÷')}
```

Comment:
- This group is lacking the `X` button. This was already pointed out. Carefully review comments by TSEs.